### ISSUE TITLE
[MAS3.2.1][Keyboard Navigation -Open bot dialog] When the user press the esc key on dialog focus is not lands on the open bot control 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1725]
**Actual Result:**
When the user presses the esc key on dialog focus is not lands on the open bot control focus is moves to the somewhere i.e bot explorer 

**Expected Result:**
When the user presses the esc key on dialog focus should be lands on the open bot control only 
When the user press the esc key on any dialog focus should be lands on that particular control only instead of moving to the some other control 

**Note:** this issue exits for all dialog controls for bot framework emulator application
**Note 2:** Same issue repro for below path 
Welcome bot Framework emulator ->Navigate to the Sign in with Azure account -> Sign in with Azure account dialog
1. Sign in with Azure account dialog
Welcome Bot Framework Emulator ->Create new bot configuration
1.Create new bot configuration
Welcome->Settings 
1.Edit ngrok settings …etc

File->All headers 
When the user press the ESC key on File menu focus is not on the File menu focus moves to the bot explorer 
Please Refer the attachment:MAS3.2.1_File.zip

**This issue also observed on MacOs, throughout the application.**

**Repro Steps:**
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the Welcome tab screen
3.	Navigate to the open bot control 
4.	Open bot dialog gets opened and Navigate to the open bot dialog using tab key  
5.	Observe the issue

**User Impact:**
Test Environment:
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}

[MAS3.2.1_Open bot dialog.zip](https://github.com/microsoft/BotFramework-Emulator/files/3500764/MAS3.2.1_Open.bot.dialog.zip)
[MAS3.2.1_File.zip](https://github.com/microsoft/BotFramework-Emulator/files/3562145/MAS3.2.1_File.zip)

[Focus should land on the same after Esc key.zip](https://github.com/microsoft/BotFramework-Emulator/files/3686450/Focus.should.land.on.the.same.after.Esc.key.zip)

**Comment by corinagum** [comment]
@Kumar, FYI I don't think the sharepoint link is working - when I click it it tries to redirect me to GitHub issues.

**Comment by Kumar2608** [comment]
@corinagum, I have updated the share point link.

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue partiall fixed .Hence we are reactivating this bug 
When we cancel the dialogs focus is moves to bot explorer control instead of respective dialog controls.
Example: When we cancel the Open bot dialog focus should stay on Open bot control only instead of moving to the Bot explorer control.

**Comment by tonyanziano** [comment]
This is related to #1876

**Comment by tonyanziano** [comment]
I believe this was actually fixed with #1876 and #1893

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version-4.5.2.-82320 Issue got fixed in "Emulator-Win32".

 But **Emulator-Mac**  is Issue still repro'ing Hence we are reactivating this bug. 
 **Test Environment:** 
OS: MacOs Version 10.15 
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)
Please refer the path :Open bot ->Close button
When the user press the enter key on close button focus is not  land on the open bot control instead of moving to the "Bot Explorer icon" but VFI is not visible until user press the tab key 
Please refer the attachment:1725.zip
[1725-Mac.zip](https://github.com/microsoft/BotFramework-Emulator/files/3706635/1725-Mac.zip)

**Comment by cwhitten** [comment]
@Kumar2608 this has been addressed on the Windows platform. Please close this issue and create a new one with the Emulator-MAC tag. Thanks!

**Comment by Kumar2608** [comment]
Issue has been fixed for win32 and Mac hence we are closing this bug.
